### DISCUSSION TITLE
Provision single server and deploy multiple sites?

### DISCUSSION THREAD

**Discussion by sutcliffe121** [deployphp/deployer#2875] [Help needed]
Does deployer support provisioning a single server and deploying multiple websites?

1x Digital Ocean Droplet
Site One: siteone.domain.com -> repo1.git
Site Two: sitetwo.domain.com -> repot2.git

I have used deployer to provision a server and deploy a single website. 

I am unsure how to deploy a second website on the same server?

I have tried to setup a 'sitetwo.domain.com' using:

`dep provision:website`

I deploy my site and it works, but the original site stops working 'siteone.domain.com'.

I have checked the docs and previous issues but can't see any useful references.

Any help is appreciated.

**Comment by antonmedv** [comment]
It should be supposed. I’ll rake a look.

**Comment by sutcliffe121** [comment]
One site works, the other displays an SSL error:
![image](https://user-images.githubusercontent.com/33236577/147918209-63ea34a1-b29c-41ef-881e-ba9499f78f55.png)

On my server, in home/deployer, I see two folders, with a Caddyfile in each.

**Reply by antonmedv** [reply]
I’ll take a look later today. Let’s open an issue.

**Reply by antonmedv** [reply]
https://github.com/deployphp/deployer/issues/2876 Please, try to debug. Maybe you will be able to detect a bug.
